modules/tensor_mechanics/src/materials/IsotropicPlasticityStressUpdate.C

```diff
@@ -24,8 +24,8 @@ InputParameters validParams<RecomputeRadialReturnIsotropicPlasticity>()
 }
 
 
-RecomputeRadialReturnIsotropicPlasticity::RecomputeRadialReturnIsotropicPlasticity(const InputParameters & parameters) :
-    RecomputeRadialReturn(parameters),
+IsotropicPlasticityStressUpdate::IsotropicPlasticityStressUpdate(const InputParameters & parameters) :
+    RadialReturnStressUpdate(parameters),
     _yield_stress_function(isParamValid("yield_stress_function") ? &getFunction("yield_stress_function") : NULL),
     _yield_stress(getParam<Real>("yield_stress")),
```

Comment:
Keeping the yield function and the hardening function below as pointers is necessary because the user can only specify one of the two (yield function or hardening function).  If there's another way to maintain this functionality with the simpler params command I'd be glad to switch